### ISSUE TITLE
Ajax Problem

### ISSUE THREAD

**Issue by metin-yilmaz** [ellisonleao/sharer.js#49]
Hello it does not work with content that comes with ajax. Will you do a work for it ?

**Comment by ellisonleao** [comment]
hey @metin-yilmaz can you maybe share a piece of code so i can check on my side?

**Comment by metin-yilmaz** [comment]
<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title></title>
    
<style type="text/css">
        body,html{margin:0; padding:0; font-size: 14px; font-family: Arial;}
        a{text-decoration: none;}
        .content{ max-width: 980px; margin:0 auto; }
        .addData{margin:20px 0; text-align: center;}
        .addData a{display: inline-block; background: #333; padding:5px; color:#fff; }
        .item{border:1px solid #bababa; background: #ccc; padding:10px; margin-bottom: 10px; }
        .social{background: #ddd; text-align: center;}
        .sharer{display: inline-block; border:none; background: black; color:#fff; cursor: pointer; padding: 10px; margin:0 2px;}
    </style>

     <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
     <script src="https://cdn.jsdelivr.net/sharer.js/latest/sharer.min.js"></script>
</head>
<body>
    
    <script type="text/javascript">
        var tempHTML = '';
        $(document).ready(function() {
            $(".addData > a").click(function(event) {
                tempHTML  += '<div class="item">';
                tempHTML  += '<div class="social">';
                tempHTML  += '<button class="sharer button" data-sharer="twitter" data-title="test!" hashtags="awesome" data-url="https://ellisonleao.github.io/sharer.js/">Share on Twitter</button>';
                tempHTML  += '<button class="sharer button" data-sharer="facebook" data-url="https://ellisonleao.github.io/sharer.js/">Share on Facebook</button>';
                tempHTML  += '<button class="sharer button" data-sharer="googleplus" data-url="https://ellisonleao.github.io/sharer.js/">Share on Google+</button>';
                tempHTML  += '</div>';
                tempHTML  += '</div>';
                $(".content").append(tempHTML);
                tempHTML = '';
            });
        });
    </script>
    <div id="wrapper">
        <div class="content">
            <div class="item">
               <div class="social">
                    <button class="sharer button" data-sharer="twitter" data-title="Checkout Sharer.js!" data-hashtags="awesome, sharer.js" data-url="https://ellisonleao.github.io/sharer.js/">Share on Twitter</button>
                    <button class="sharer button" data-sharer="facebook" data-url="https://ellisonleao.github.io/sharer.js/">Share on Facebook</button>
                    <button class="sharer button" data-sharer="googleplus" data-url="https://ellisonleao.github.io/sharer.js/">Share on Google+</button>
               </div>
            </div>
        </div>
        <div class="addData"><a href="javascript:;">Add New Data</a></div>
    </div>
</body>
</html>

**Comment by metin-yilmaz** [comment]
Sample example code i shared. share links do not work when added to the page later and they don't work dynamically also(ajax, javascript after loaded)

**Comment by ellisonleao** [comment]
Hey @metin-yilmaz i've fixed this issue on the last version. I've updated your code showing how to add a share behaviour on ajax calls here http://codepen.io/ellisonleao/pen/XRJdYV

Please let me know if this works for you.

**Comment by metin-yilmaz** [comment]
I have tested and it works very well. thank you so much 👍
